functions/download_recount2_data.R

```diff
@@ -0,0 +1,124 @@
+## Run this once to setup environment
+## Used R 3.6.3
+#if (!requireNamespace("BiocManager", quietly = TRUE))
+#  install.packages("BiocManager")
+
+#BiocManager::install("recount")
+
+## About
+# recount2 contains over 70,000 uniformly processed human RNA-seq samples spanning
+# TCGA, SRA and GTEx
+# library('recount')
```

Comment:
Don't the functions you're using depend on `recount` being imported?

Reply:
Recount is actually imported in the notebook. I assume either would work...